config/config.php

```diff
@@ -1,33 +1,35 @@
 <?php
 
-use Zend\Stdlib\ArrayUtils;
-use Zend\Stdlib\Glob;
+use Zend\ConfigAggregator\ConfigAggregator;
+use Zend\ConfigAggregator\PhpFileProvider;
 
-/**
- * Configuration files are loaded in a specific order. First ``global.php``, then ``*.global.php``.
- * then ``local.php`` and finally ``*.local.php``. This way local settings overwrite global settings.
- *
- * The configuration can be cached. This can be done by setting ``config_cache_enabled`` to ``true``.
- *
- * Obviously, if you use closures in your config you can't cache it.
- */
+$cacheConfig = [
+    'config_cache_path' => 'data/config-cache.php',
+];
 
-$cachedConfigFile = 'data/cache/app_config.php';
+$aggregator = new ConfigAggregator(
+    [
```

Comment:
Start the array with the opening paren on the previous line, and use the following structure:

```php
$aggregator = new ConfigAggregator([
    // entries go here
], $cacheConfig['config_cache_path']);
```

This will do the following:

- Reduce indentation.
- Make it more clear that the caching flag is a separate argument from the aggregator entries themselves.

Thanks!